src/fields/multi_field.impl.hh

```diff
@@ -58,6 +59,17 @@ it::Array MultiField<spacedim,Value>::get_multifield_input_type() {
 
 
 template<int spacedim, class Value>
+auto MultiField<spacedim, Value>::disable_where(
+        const MultiField<spacedim, typename FieldValue<spacedim>::Enum > &control_field,
+        const vector<FieldEnum> &value_list) -> MultiField &
+{
+    no_check_control_field_=&control_field;
```

Comment:
Could we rather call disable_where for individual subfields? We should also check that the control_field have same size.
This way we save redundant member no_check_control_field_ of the MultiField.